notebooks/EOPF_Tutorials/EOPF_SAFE_ZARR_COMPARE_METADATA_SENTINEL2_L2A.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    file_path = "/mnt/CEPH_PROJECTS/test/S2B_MSIL2A_20250206T101109_N0511_R022_T32TPS_20250206T122414.SAFE.zip"
</pre><p>Please create a new GitHub issue asking EODC to upload this SAFE/zip to their bucket so that the notebook can be fully reproducible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EOPF-Sample-Service/eopf-sample-notebooks/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>